lib/event-highlighting/event-highlight-worker.js

```diff
@@ -0,0 +1,40 @@
+const EventMapHelper = require('./event-map-helper')
+
+require("../polyfills/set");
+
+let messageBuffer = new Map();
+let receivedThisFrame = new Map();
+
+self.onmessage = function(e) {
+  const event = e.data;
+
+  queueEvent(event);
+};
+
+setInterval(() => {
+    const {active, added, removed} = EventMapHelper.diffEventMaps(
+      messageBuffer,
+      receivedThisFrame,
+    );
+
+    postMessage({active, added, removed });
+    messageBuffer = EventMapHelper.transformedEvents(added.union(active));
+    receivedThisFrame.clear();
+
+}, 1000/30);
```

Comment:
magic number, could it be extracted in a const with a meaningful name?

Reply:
I added a const for this. I wasn't sure how I wanna handle this. Now the animation is locked by a maximum of 30FPS. Maybe I will make it configurable at a later point again, but the web worker has no access of the atom config properties, because they are only available within the main thread.

Reply:
Done